easy-ui-react/src/RadioButtonGroup/RadioButtonGroup.module.scss

```diff
@@ -45,4 +45,9 @@
     color: design-token("color.neutral.600");
     border-color: design-token("color.neutral.300");
   }
+
+  &[data-focus-visible] {
+    @include native-focus-ring;
+    z-index: 3;
```

Comment:
`z-index` update so that the outline is always above the border of the neighboring elements.
<img width="368" alt="Screenshot 2024-12-17 at 4 58 45 PM" src="https://github.com/user-attachments/assets/2ea07bd4-f1c0-4a93-8551-fc20f6ec6f79" />

Reply:
Without:
<img width="343" alt="Screenshot 2024-12-17 at 4 59 30 PM" src="https://github.com/user-attachments/assets/17af71ac-27c9-4bca-89ff-86949f968167" />